src/containers/coursePageDetails/index.jsx

```diff
@@ -42,106 +70,118 @@ class coursePageDetails extends Component {
-
-                  <div className={styles.stars}>
-                    <StarRatings
-                      rating={review.rating}
-                      starRatedColor="#ff8c05"
-                      starEmptyColor="#515151"
-                      starDimension="24px"
-                      starSpacing="3px"
-                      numberOfStars={5}
-                      name={`review${review.key}`}
-                    />
+              <header className={styles.headerWrapper}>
+                <div className={styles.headerDescription}>
+                  <h2 className={styles.courseTitle}>{courseName}</h2>
+                  <p className={styles.shortDescription}>{description}</p>
+                  <div className={styles.courseStats}>
+                    <div className={styles.stars}>
+                      <StarRatings
+                        rating={rating}
+                        starRatedColor="#ff8c05"
+                        starEmptyColor="#515151"
+                        starDimension="24px"
+                        starSpacing="3px"
+                        numberOfStars={5}
+                        name="courseRating"
+                      />
+                    </div>
+                    <span>{rating}</span>
+                    <span className={styles.verticalLine} />
+                    <span>
+                      <font>{translate('Students')}: </font> {numberOfEnrolledStudents}
+                    </span>
                   </div>
-
-                  <span>{review.createdAt}</span>
+                  <div>
+                    <span>
+                      <font>{translate('Author')}: </font>
+                      {authors}
+                    </span>
+                    <span className={styles.verticalLine} />
+                    <span>
+                    <font>{translate('Language')}: </font>
+                      {language}
+                    </span>
+                  </div>
+                </div>
+                {
+                  isAuthenticated && !subscribed &&
+                  (<div className={styles.subscribeButtonWrapper}>
+                    <button className={styles.subscribeButton} type='button' onClick={this.subscribeUserCourse}>{translate('Subscribe')}</button>
+                  </div>)
+                }
+                {
+                  isAuthenticated && subscribed && (
+                    <div className={styles.subscribeButtonWrapper}>
+                      <span className={styles.subscribeNotification}>{translate('You have already subscribed')}</span>
+                    </div>
+                  )
+                }
+              </header>
+
+              <hr />
+              <p className={styles.profitTitle}>{translate('Your profits')}</p>
+              <ul className={styles.profitList}>
+                {profits && profits.map(el => (
+                  <li key={el.id}>{el.description}</li>
+                ))}
```

Comment:
Use sensitive variables in array methods:
`el => profit` in your case

As I see, profits is an array, so your check isn't right, cause empty array will also return true.

Reply:
Why didn't you say that to Sergey???

Reply:
Yes, you are right that `Boolean([])` => `true` and then you try to get object field `profit` in empty object `{}`

Reply:
`[] => true`, `[....] => true`. Anyway you'll get true, isn't it? You should check arrays like `profits.length !== 0`

Reply:
In the initial state, We haven't any array.

Reply:
```
state = {
    course: {},
    error: null,
    showReviewsNum: 1,
    isLoading: true,
    subscribed: false,
  };
```